pystruct/learners/n_slack_ssvm.py

```diff
@@ -12,13 +12,17 @@
 import cvxopt
 import cvxopt.solvers
 
-from sklearn.externals.joblib import Parallel, delayed
+from sklearn.externals.joblib import Parallel, delayed, cpu_count
 from sklearn.utils import gen_even_slices
 
 from .ssvm import BaseSSVM
 from ..utils import unwrap_pairwise, find_constraint
 
 
+def find_constraint_map(args):
+    return find_constraint(* args)
```

Comment:
Ah. I'm surprised that doesn't work. Then maybe just define a mapstar?
how about
```python
def mapstar(func, list):
    def apply_starred(stuff):
        func, args = stuff
        return func(*args)
    map(apply_starred, ((func, x) for x in list))
```
?